### DISCUSSION TITLE
Should I enable `FlushConsolidationHandler` by default?

### DISCUSSION THREAD

**Discussion by He-Pin** [netty/netty#13608] [General]
When migrating pekko from Netty 3 to Netty 4, I found the `write` will need change to `writeAndFlush`, because the SPI of the classical transport will not call any `flush()`(Not defined.)

In Netty 4, which will not do autoflush, but now many `writeAndFlush` will hurt performance too.

Should I add a `FlushConsolidationHandler` in the pipeline or add it behind some config? what's the best default config?

I read part of the sourcecode, which seems will not introduce much latency, am I right，and any suggestion, thanks.

The line is :https://github.com/apache/incubator-pekko/pull/643/files#diff-fed1e8c75aa717d96d8f5038c571f142eba61fb9fdcfa9b4e8be52add74c0e63R120

**Comment by chrisvest** [comment]
I think using the `FlushConsolidationHandler` is a fair compromise when upgrading from Netty 3 to 4. Ideally you want to be explicit about  when you `write` and when you `flush`, but making this change to a big code base can be tedious and error prone.
The `FlushConsolidationHandler` let you finish the code migration sooner, and then straighten out the `write` and `flush` calls later, as a series of smaller changes.

**Reply by He-Pin** [reply]
Thanks， what's the default parameter would you suggest:)?
